README.md

```diff
@@ -1 +1 @@
-# code-review
\ No newline at end of file
+# Product-landing-page
```

Comment:
It is good that you added a descriptive title but there's more that needs to be done in order to make this a descriptive readme file. 
Add more fields e.g acknowledgements, authors, you can use [this template](https://github.com/microverseinc/readme-template)

Reply:
Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions